bertopic/_utils.py

```diff
@@ -147,3 +147,57 @@ def validate_distance_matrix(X, n_samples):
         raise ValueError("Distance matrix cannot contain negative values.")
 
     return X
+
+
+def select_topic_representation(
+    ctfidf_embeddings: Optional[Union[np.ndarray, csr_matrix]] = None,
+    semantic_embeddings: Optional[np.ndarray] = None,
+    use_ctfidf: bool = True
+) -> tuple[np.ndarray, bool]:
+    """Select the topic representation.
+
+    Arguments:
+        ctfidf_embeddings: The c-TF-IDF embedding matrix
+        semantic_embeddings: The semantic embedding matrix
+        use_ctfidf: Whether to use the c-TF-IDF representation. If False, topics embedding representation is used, if it
+                    exists. Default is True.
+    Raises
+        ValueError:
+            - If no topic representation was found
+            - If c-TF-IDF embeddings are not a numpy array or a scipy.sparse.csr_matrix
+
+    Returns:
+        The selected topic representation and a boolean indicating whether it is c-TF-IDF.
+    """
+
+    def to_ndarray(array: Union[np.ndarray, csr_matrix]) -> np.ndarray:
+        if isinstance(array, csr_matrix):
+            return array.toarray()
+        elif not isinstance(array, np.ndarray):
+            raise ValueError("The embeddings should be either a type of `numpy.ndarray` or `scipy.sparse.csr_matrix`")
+        return array
+
+    logger = MyLogger("WARNING")
```

Comment:
Just checking. If the logger is initialized outside of BERTopic, does the general logging still works? Even if you instantiate BERTopic multiple times?

Reply:
Good catch. 

Since `getLogger(..)` returns just a reference, then there should no issue. The problem is with the repetitive setting of the level and possibly setting of other parameters. 

I would propose to create create a new method of MyLogger called `configure()` where the configuration takes place:
```
self.set_level(level)
self._add_handler()
self.logger.propagate = False
```

This would be called only once. 

Then, it would be possible to call anywhere MyLogger() and get the correct logger. What do you think?

Reply:
That sounds good considering the logger needs to be created only once and then retrieved a couple of times throughout the code.

Reply:
Ok. Done.